js/app.js

```diff
@@ -126,14 +126,15 @@ function randomizeClearings(gameMap) {
   return availableClearings
     .map(clearing => ({ sortIndex: Math.random(), value: clearing }))
     .sort((a, b) => a.sortIndex - b.sortIndex)
-    .map(sortableClearing => sortableClearing.value)
-    .splice(0, gameMap.numClearings);
```

Comment:
Is removing this intentional? It means the function no longer properly supports maps that have a number of clearings which is not a multiple of 3.